src/workshop/coffe-script.ts

```diff
@@ -0,0 +1,30 @@
+const MY_NAME: string = "Leena"
+let myMilkAmount: number = 50
+let myCoffeType: string = "cappuccino"
+
+console.log(`Hello, ${MY_NAME}! Your coffe is ${myCoffeType} and it has ${myMilkAmount} ml.`)
+console.log(`Hello, ` + MY_NAME + `! Your coffe is ${myCoffeType} and it has ${myMilkAmount} ml.`)
```

Comment:
You may use ' ' quotes instead of ` ` in concatenation

Reply:
will not change for now.